packages/launch/src/focused-launch/summary/style.scss

```diff
@@ -82,20 +82,12 @@ $commentary-column-gap-wide: 100px;
 	@include break-wide {
 		gap: $commentary-column-gap-wide;
 	}
-}
 
-// by default: highlight all steps when nothing is in focus
-.focused-launch-container:not( :focus-within ) {
-	.focused-launch-summary__step {
+	&.highlighted {
```

Comment:
Probably a shorter `transition-duration` of `0.2s` would make the animation feel a bit snappier? (see line 76 in this same file)